README.md

```diff
@@ -125,6 +125,16 @@ Add `READ_PROFILE` and/or `WRITE_PROFILE` permissions to `AndroidManifest.xml`
 ...
 ```
 
+##### ProGuard
```

Comment:
Thank you for your PR. This section should actually go on line 99, under Installation > Manual > Android > Proguard

Reply:
Done